thoughtis/cata-blocks
# Issue#76/toc open

### Related Issues
- Resolved #76 

### What Was Accomplished
- Added three levels of control for the Open/Close Behavior of a Table of Contents
  - Always Open
  - Start Open
  - Start Closed
- The "Always Open" setting uses `div` elements instead of `details` and `summary`. Other options for keeping the TOC open like adding `onclick="return false;"` removed default functionality. This way should be more accessible since the elements behave as expected.

### How It Was Tested
- [x] Local plugin dev site
- [x] Local theme dev site
- [x] Remote theme dev site

### How To Test

Create a table of contents in a post using each of these earlier versions, then switch to this branch and ensure Attempt Block Recovery works.

- [x] A version tag 0.7.5 or earlier
- [x] Main branch
